README.md

```diff
@@ -1,94 +1,157 @@
-package that automates every aspect of an application's life.
+> - `ops` is  [available on PyPI](https://pypi.org/project/ops/).
+> - The latest version of `ops` requires Python 3.8 or above.
 
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
+||||
+|-|-|- |
+|| [Juju](https://juju.is/docs/juju) | Learn how to quickly deploy, integrate, and manage charms on any cloud with Juju. <br>  _It's as simple as `juju deploy foo`, `juju integrate foo bar`, ..., on any cloud._ |
+||||
+|| [Charmhub](https://charmhub.io/) | Sample our existing charms on Charmhub. <br> _A charm can be a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a data platform ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more._ |
+||||
+|:point_right:| [Charm SDK](https://juju.is/docs/sdk) | Write your own charm! <br> _Juju is written in Go, but our SDK supports easy charm development in Python._  |
 
-Charms should do one thing and do it well. Each charm drives a single
-application and can be integrated with other charms to deliver a complex
-system. A charm handles creating the application in addition to scaling,
-configuration, optimisation, networking, service mesh, observability, and other
-day-2 operations specific to the application.
+## Give it a try
 
-The ops library is part of the Charm SDK (the other part being Charmcraft).
-Full developer documentation for the Charm SDK is available at
-https://juju.is/docs/sdk.
+Let's use `ops` to build a Kubernetes charm:
 
-To learn more about Juju, visit https://juju.is/docs/olm.
+### Set up
 
+> See [Charm SDK | Set up your development environment automatically > Set up an Ubuntu `charm-dev` VM with Multipass](https://juju.is/docs/sdk/dev-setup#heading--automatic-set-up-an-ubuntu-charm-dev-vm-with-multipass). <br> Choose the MicroK8s track. 
 
-## Pure Python
 
-The framework provides a standardised Python object model that represents the
-application graph, as well as an event-handling mechanism for distributed
-system coordination and communication.
+### Write your charm
 
-The latest version of ops requires Python 3.8 or above.
+On your Multipass VM, create a charm directory and use Charmcraft to initialise your charm file structure:
 
-Juju itself is written in Go for efficient concurrency even in large
-deployments. Charms can be written in any language, however, we recommend using
-Python with this framework to make development easier and more standardised.
-All new charms at Canonical are written using it.
+```
+mkdir my-new-charm
+cd my-new-charm
+charmcraft init
+```
+This has created a standard charm directory structure. Poke around.
```

Comment:
Instead of three paragraphs following this that start with "Note that...", we could just remove the "Note that". Or append "Things to note:" to this paragraph and then make the notes bullet points (without the "note that" suffix).

Reply:
Done.